.eslintrc

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Is the .js file no longer supported? I don't think so.. Why the move to a json file?